packages/cli/lib/services/init.service.unit.cli-test.ts

```diff
@@ -0,0 +1,43 @@
+import { expect, describe, it, afterEach, vi } from 'vitest';
```

Comment:
Does it really need to be `cli-test` (vs unit.test)?

Reply:
Definitely not - was just following the prior pattern for where those files were. Will fix.

Reply:
Fixed in 704bf3c92